alws/crud/errata.py

```diff
@@ -1757,5 +1755,37 @@ async def reset_matched_erratas_packages_threshold(
             await prepare_resetting(items_to_insert, record, session)
         session.add_all(items_to_insert)
         await session.flush()
-    logging.info(f'Packages for records {[record.id for record in records]}'
-                 f' have been matched if their date is later than {issued_date}')
+    logging.info(
+        f'Packages for records {[record.id for record in records]}'
+        f' have been matched if their date is later than {issued_date}'
+    )
+
+
+async def set_errata_packages_in_progress(records_ids: List[str]):
+    async with open_async_session(key=get_async_db_key()) as session:
```

Comment:
Could you please use `alws.dependencies.get_async_db_session` for session opening instead?

Reply:
done